YIF.Core.Domain/ServiceInterfaces/IIoEAdminService.cs

```diff
@@ -11,6 +11,7 @@ public interface IIoEAdminService
         Task<ResponseApiModel<DescriptionResponseApiModel>> AddSpecialtyToIoe(SpecialtyToInstitutionOfEducationPostApiModel specialtyToIoE);
         Task DeleteSpecialtyToIoe(SpecialtyToInstitutionOfEducationPostApiModel specialtyToIoE);
         Task<ResponseApiModel<DescriptionResponseApiModel>> UpdateSpecialtyDescription(SpecialtyDescriptionUpdateApiModel specialtyDescriptionUpdateApiModel);
+        Task<ResponseApiModel<SpecialtyToInstitutionOfEducationResponseApiModel>> GetSpecialtyToIoEDescription(string userId, string specialtyId);
```

Comment:
Sorry, but i think that parameter specialtyId shoud be renamed to specialtyToIoEId, because it's easier not to think about business rules when reading the code.